index.js

```diff
@@ -1,93 +1,53 @@
-const fs = require("fs"),
-    http = require("http"),
-    path = require("path"),
-    methods = require("methods"),
-    express = require("express"),
-    bodyParser = require("body-parser"),
-    session = require("express-session"),
-    cors = require("cors"),
-    passport = require("passport"),
-    errorhandler = require("errorhandler"),
-    mongoose = require("mongoose");
+import express from 'express';
+import bodyParser from 'body-parser';
+import swaggerJSDoc from 'swagger-jsdoc';
+import swaggerui from 'swagger-ui-express';
+//import swaggerDoc from './docs/swagger';
 
-const isProduction = process.env.NODE_ENV === "production";
 
 // Create global app object
 const app = express();
 
-app.use(cors());
+// swagger definition
```

Comment:
Can you define the swagger configuration in a separate file and add it to a config folder

Reply:
Also, add sample documentation, so your peers know how to you it